docs/proposed/http-storage-node-protocol.rst

```diff
@@ -382,6 +382,11 @@ the server will respond with ``400 BAD REQUEST``.
 
 If authorization using the secret fails, then a ``401 UNAUTHORIZED`` response should be sent.
 
+Encoding
+~~~~~~~~
+
+* ``storage_index`` should be base32 encoded (RFC3548) in URLs.
```

Comment:
It was motivated by their existence, yes. But easy enough to use something else, I don't have a strong opinion, so would be good to know which you prefer.

Reply:
I'm not sure.  Absent other considerations, base64url seems better because it's more compact and because we've started adopting it elsewhere.

On the other hand, consistency with other encoding of storage-index in the codebase doesn't seem like a bad thing.  Storage indexes aren't the *most* user-facing thing but they do show up here and there ... and for an admin you're probably going to run into them.  Having to deal with two different formats seems like it would be a drag.

Reply:
Consistency would be good here when debugging any differences between this and Foolscap (right?).

Reply:
OK, leaving unchanged.